assets/js/base/components/tabs/index.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { find, partial } from 'lodash';
+import { withInstanceId } from '@wordpress/compose';
+import classnames from 'classnames';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const TabButton = ( {
+	tabId,
+	onClick,
+	children,
+	selected,
+	ariaLabel,
+	...rest
+} ) => {
+	return (
+		<button
+			role="tab"
+			type="button"
+			tabIndex={ selected ? null : -1 }
+			aria-selected={ selected }
+			aria-label={ ! ariaLabel ? null : ariaLabel }
+			id={ tabId }
+			onClick={ onClick }
+			{ ...rest }
+		>
+			<span className="wc-component__tab-item-content">{ children }</span>
+		</button>
+	);
+};
+
+const Tabs = ( {
+	className,
+	onSelect = () => null,
+	tabs,
+	activeClass = 'is-active',
+	initialTabName,
+	instanceId,
+	ariaLabel = __( 'Tabbed Content', 'woo-gutenberg-products-block' ),
+	children,
+} ) => {
+	const [ selected, setSelected ] = useState( '' );
```

Comment:
Oh ya, that's inconsistent.  I started with a string because it keeps things the same type so I probably should set to an empty string instead of null later too.  Good catch!